spec/controllers/rooms_controller_spec.rb

```diff
@@ -90,4 +90,25 @@
       expect(response).to have_http_status(:created)
     end
   end
+
+  describe '#recordings' do
+    it 'ids of recordings that belong only to room with given room_friendly_id' do
+      room1 = create(:room, user:, friendly_id: 'friendly_id_1')
+      room2 = create(:room, user:, friendly_id: 'friendly_id_2')
+      recordings = create_list(:recording, 5, room: room1)
+      create_list(:recording, 5, room: room2)
+      get :recordings, params: { room_friendly_id: room1.friendly_id }
+      recording_ids = JSON.parse(response.body)['data'].map { |recording| recording['id'] }
+      expect(response).to have_http_status(:ok)
+      expect(recording_ids).to eq(recordings.pluck(:id))
```

Comment:
Really nicely done though

Reply:
room2 isn't used?
Is it part of the test?
line 97 and 99

Reply:
Would it be a valid test if we remove room2 (line 97 and 99) as well?

Reply:
Yes it would still pass, but the point of them is to show that even though there are recordings for other rooms that exist in the db, the request only returns the recordings that belong to the room with the specific friendly_id given in the request.